packages/components/src/search-list-control/index.js

```diff
@@ -0,0 +1,311 @@
+/**
+ * External dependencies
+ */
+import { __, _n, sprintf } from '@wordpress/i18n';
+import {
+	Button,
+	MenuGroup,
+	Spinner,
+	TextControl,
+	withSpokenMessages,
+} from '@wordpress/components';
+import { Component, Fragment } from '@wordpress/element';
+import { compose, withInstanceId, withState } from '@wordpress/compose';
+import { escapeRegExp, findIndex } from 'lodash';
+import Gridicon from 'gridicons';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { buildTermsTree } from './hierarchy';
+import SearchListItem from './item';
+import Tag from '../tag';
+
+const defaultMessages = {
+	clear: __( 'Clear all selected items', 'woo-gutenberg-products-block' ),
```

Comment:
Should the text domain change to `wc-admin` throughout all the files in the PR?

Reply:
Yeah, my mistake – fixed it 👍